### PR TITLE
Fixed most warnings in text example.

### PR DESCRIPTION
**Pull request by azoitl**

### CODE REVIEW

File: org.eclipse.gef.examples.text/src/org/eclipse/gef/examples/text/model/commands/CompoundTextCommand.java [hunk 1]
```diff
@@ -78,8 +76,7 @@ public void execute() {
 
 	@Override
 	public void executePending() {
-		for (int i = 0; i < pending.size(); i++) {
-			Command cmd = (Command) pending.get(i);
+		for (Command cmd : applied) {
```
Comment (ptziegler) [inline org.eclipse.gef.examples.text/src/org/eclipse/gef/examples/text/model/commands/CompoundTextCommand.java:79]:
```suggestion
		for (Command cmd : pending) {
```

Comment (ptziegler) [reply org.eclipse.gef.examples.text/src/org/eclipse/gef/examples/text/model/commands/CompoundTextCommand.java:79]: I think history has shown that it's usually not a good idea to convert for loops into for-each loop without checking whether it's impossible for a ConcurrentModifcationException to occur.
Would it make sense to use CopyOnWriteArrayLists for something as fundamental as a compound command?

Comment (azoitl) [reply org.eclipse.gef.examples.text/src/org/eclipse/gef/examples/text/model/commands/CompoundTextCommand.java]: Ah just learned that I could have applied your suggested change directly here. Did it locally and force pushed.

Regarding your for-each loop. Yes you are right. However the command that we are talking here about is a command which is currently only within the text example. Furthermore it is not even used there. So I think it is fine to have it like that. Furthermore the text example would anyhow need some love as some of its features are currently broken.

Comment (azoitl) [reply org.eclipse.gef.examples.text/src/org/eclipse/gef/examples/text/model/commands/CompoundTextCommand.java]: But that is some work for another day.

Comment (ptziegler) [reply org.eclipse.gef.examples.text/src/org/eclipse/gef/examples/text/model/commands/CompoundTextCommand.java]: That's fine by me. It also looked good to me as well so I think we can go with how it is right now.

**Pull request merged**